docs/hub/api.md

```diff
@@ -2,25 +2,226 @@
-| /api/repos/move POST                                                         	| Move a repository (rename within same namespace or transfer from user to organization).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 	| `move_repo()`                     	| ```headers = { "authorization" :  "Bearer $token" }```    ```json= {"fromRepo" : "namespace/repo_name", "toRepo" : "namespace2/repo_name2"}```                        	|   	|
-| /api/whoami-v2    GET                                                           	| Get username and organizations the user belongs to.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    	| `whoami(token)`                   	| ```headers = { "authorization" :  "Bearer $token" }```                                                                                                                	|   	|
+The base URL for those endpoints below is `https://huggingface.co`. For example, to construct the `/api/models` call below, one can call the URL [https://huggingface.co/api/models](https://huggingface.co/api/models)
+
+## GET `/api/models`
```

Comment:
I would remove the backticks because the code formatting isn't rendered in the right navbar so it looks a little messy 😅

```suggestion
## GET /api/models
```

Reply:
@stevhliu is that something we could fix in docs-builder instead? (showing the code formatting correctly)

Reply:
but i think it would look better without the code formatting anyways, no? can we try removing them to see what it looks like?

Reply:
(just pushed https://github.com/huggingface/hub-docs/pull/962/commits/917a37ecb85e91b7b028d2e6c4164695b68fd6e0)

Reply:
when you reping here again when the preview's fixed? thanks! (cc @mishig25 )

Reply:
@julien-c it's fixed. New version of the PR docs is here: https://moon-ci-docs.huggingface.co/docs/hub/pr_962/en/api

Reply:
Ok so personally i think it works like that, WDYT @osanseviero ?

Reply:
Yes, I agree this works!